.buildkite/scripts/steps/upload_to_s3.sh

```diff
@@ -12,8 +12,8 @@
 # Experience indicates that BuildKite always sets BUILDKITE_PULL_REQUEST to
 # be the PR number or "false". Hence we explicitly check for "false" here.
 if [[ x"$BUILDKITE_PULL_REQUEST" = xfalse && -z "$ML_DEBUG" ]] ; then
-    . dev-tools/aws_creds_from_vault.sh
-    echo 'Uploading artefacts to S3'
+    . .buildkite/scripts/common/aws_creds_from_vault.sh
+    echo 'Uploading artifacts to S3'
     ./gradlew --info -Dbuild.version_qualifier=$VERSION_QUALIFIER -Dbuild.snapshot=$BUILD_SNAPSHOT upload
```

Comment:
This only uploads platform-specific artifacts. It should be easy to create an `uploadAll` task that uploads everything in one go by duplicating this task and changing the pattern used in the new task: https://github.com/elastic/ml-cpp/blob/45a215a01faff24971505f3aa99c88d32d6c2ac3/build.gradle#L437

Reply:
Do we need this file at all? `upload_dra_to_s3.sh` uploads everything to S3 at the end of the build, and there's no need to upload intermediate artifacts now because we're using BuildKite functionality to transfer artifacts between steps. So probably this file can be deleted and the only file that needs to touch S3 is `upload_dra_to_s3.sh`.